services/ingestion/etl/types/Project.js

```diff
@@ -11,10 +11,17 @@ type Geolocation = {
   lon: string,
 };
 
+type GeoJSON = {
+  type: string,
+  coordinates: Array<Geolocation>,
```

Comment:
The `Geolocation` type is obsolete. `coordinates`is an array of:
- 2 numbers (or 3 if we consider the altitude)
- arrays of 2 numbers (or 3 if we consider the altitude)
- arrays of arrays of 2 numbers (or 3 if we consider the altitude)
:D